advanced-list/src/app.js

```diff
@@ -2,6 +2,7 @@ import xs from 'xstream';
 import {h3, div} from '@cycle/dom';
 import isolate from '@cycle/isolate';
 import Ticker from './Ticker.js';
+import Collection from 'cycle-collections';
```

Comment:
If we put this package in `@cycle` org, we could name it `@cycle/collection`. I don't see how pluralizing would help since it's `import Collection` and not `import Collections`, and we don't have any other package named with plural.